model/qti/AssetParser.php

```diff
@@ -105,9 +104,22 @@ public function extract(){
                 $this->extractXinclude($element);
             }
         }
+        $this->extractApipAccessibilityAssets();
         return $this->assets;
     }
 
+    private function extractApipAccessibilityAssets()
+    {
+        if (property_exists($this->item, 'apipAccessibility')) {
+            try {
+                $assets = tao_helpers_Xml::extractElements('fileHref', $this->item->getApipAccessibility(), 'http://www.imsglobal.org/xsd/apip/apipv1p0/imsapip_qtiv1p0');
```

Comment:
psr 2.3

Reply:
?

Reply:
@yuryfilippovich could you explain me what don't you like here and from when TAO supports psr2.3?

Reply:
limit for one line - 120 characters. look like codeclimate find the same issue

Reply:
fixed